### PR TITLE
adds tags for mip-rna

### PR DESCRIPTION
**Pull request by jemten**
### This PR adds | fixes:
- Adds hk tags for MIP rna
- Adds deepvariant tags
- Use markduplicates tags for storing cram
- Expand mip cli into mip-dna and mip-rna cli

**How to prepare for test**:
- [x] `ssh` to hasta.scilfelab.se
- [x] Install on stage:
  - `conda activate S_hermes`
  - `pip install -U "git+https://github.com/clinical-genomics/hermes@mip-rna" --force-reinstall --no-deps`
  - `pip install -U "git+https://github.com/clinical-genomics/hermes@mip-rna"`

### How to test:
- Will be tested with https://github.com/Clinical-Genomics/cg/pull/1151

### Review:
- [x] Code approved by MM, MR
- [x] Tests executed by AJ, HS
- [x] "Merge and deploy" approved by HS, MM

This [version](https://semver.org/) is a:
- [x] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Deploy this branch with: https://github.com/Clinical-Genomics/cg/pull/1151

### CODE REVIEW

**Comment by jemten** [comment]
Merging this should be coordinated with the coming MIP update. Would appreciate an initial review from @moonso or @Mropat so that I can see if changes are required in MIP's deliverables file as well.

**Comment by Mropat** [comment]
:fireworks:  :man_dancing: :3rd_place_medal:  :champagne:

**Review by Mropat — APPROVED** [review]

File: cg_hermes/config/tags.py [hunk 1]
```diff
@@ -31,6 +31,7 @@
     "somatic": {"description": "Associated with somatic variants"},
     "rhocall-viz": {"description": "Runs of homozygosity index"},
     "smn-calling": {"description": "Copy number calls for the SMN gene"},
+    "snv": {"description": "Single nucleotide variants and short indels"},
```
Comment (moonso) [inline cg_hermes/config/tags.py:34]: hehe funny that did not exist

File: cg_hermes/deliverables.py [hunk 2]
```diff
@@ -76,10 +77,15 @@ def set_pipeline_specific_variables(self):
             self.files = self.get_mutant_files()
             self.configs = Deliverables.build_internal_tag_map(MUTANT_COMMON_TAGS)
         elif self.pipeline == Pipeline.MIP_DNA:
-            LOG.info("Parsing deliverables for mip")
+            LOG.info("Parsing deliverables for mip_dna")
```
Comment (moonso) [inline cg_hermes/deliverables.py:80]: Could you remove this and add one logging message after the if else:

```python
LOG.info("Parsing deliverables for %s", self.pipeline)
```

**Review by moonso — APPROVED** [review]
Looks great!

File: cg_hermes/deliverables.py (hunk 2 above)
Comment (henrikstranneheim) [reply cg_hermes/deliverables.py]: Done

File: cg_hermes/config/tags.py (hunk 1 above)
Comment (henrikstranneheim) [reply cg_hermes/config/tags.py:34]: 😄

**Comment by henrikstranneheim** [comment]
Deployed on hasta:
```
[0|0|983] [henrik.stranneheim@hasta:~/git/hermes] [P_main] main ± cg deploy hermes
Running CG DEPLOY
Set dry run to False
Deploying hermes with CG
Deploying the hermes software
Use app config /home/proj/production/servers/config/hasta.scilifelab.se/hermes-deploy-prod.yaml
Running command /home/proj/bin/conda/envs/P_shipping/bin/shipping --host-config /home/proj/production/servers/config/hasta.scilifelab.se/host_config_prod.yaml --app-config /home/proj/production/servers/config/hasta.scilifelab.se/hermes-deploy-prod.yaml deploy
2021-06-03 14:19:33 hasta.scilifelab.se shipping.cli.base[21459] INFO Hello shipper henrik.stranneheim
2021-06-03 14:19:33 hasta.scilifelab.se shipping.cli.base[21459] INFO Sailing on hasta.scilifelab.se
2021-06-03 14:19:33 hasta.scilifelab.se shipping.cli.base[21459] INFO Use config /home/proj/production/servers/config/hasta.scilifelab.se/hermes-deploy-prod.yaml
2021-06-03 14:19:33 hasta.scilifelab.se shipping.cli.deploy[21459] INFO Running shipping deploy
2021-06-03 14:19:33 hasta.scilifelab.se shipping.environment[21459] DEBUG Found conda env path: /home/proj/bin/conda/envs/P_shipping
2021-06-03 14:19:33 hasta.scilifelab.se shipping.environment[21459] INFO Locating base to /home/proj/bin/conda
2021-06-03 14:19:33 hasta.scilifelab.se shipping.commands[21459] DEBUG Initialising Process with binary: /home/proj/bin/conda/envs/P_hermes/bin/python
2021-06-03 14:19:33 hasta.scilifelab.se shipping.commands[21459] DEBUG Use base call ['/home/proj/bin/conda/envs/P_hermes/bin/python']
2021-06-03 14:19:33 hasta.scilifelab.se shipping.package[21459] INFO Fetch current version for package cg-hermes
2021-06-03 14:19:33 hasta.scilifelab.se shipping.commands[21459] INFO Running command /home/proj/bin/conda/envs/P_hermes/bin/python -m pip show cg-hermes
2021-06-03 14:19:35 hasta.scilifelab.se shipping.cli.deploy[21459] INFO henrik.stranneheim wants to deploy cg-hermes on host hasta.scilifelab.se in environment P_hermes
2021-06-03 14:19:35 hasta.scilifelab.se shipping.deploy.conda[21459] INFO Check if conda is available on your system
2021-06-03 14:19:35 hasta.scilifelab.se shipping.environment[21459] DEBUG Found conda env path: /home/proj/bin/conda/envs/P_shipping
2021-06-03 14:19:35 hasta.scilifelab.se shipping.environment[21459] INFO Locating base to /home/proj/bin/conda
2021-06-03 14:19:35 hasta.scilifelab.se shipping.environment[21459] DEBUG Conda is available: /home/proj/bin/conda
2021-06-03 14:19:35 hasta.scilifelab.se shipping.deploy.conda[21459] INFO You are in conda environment P_main
2021-06-03 14:19:35 hasta.scilifelab.se shipping.environment[21459] DEBUG Found conda env path: /home/proj/bin/conda/envs/P_shipping
2021-06-03 14:19:35 hasta.scilifelab.se shipping.environment[21459] INFO Locating base to /home/proj/bin/conda
2021-06-03 14:19:35 hasta.scilifelab.se shipping.deploy.conda[21459] INFO You want to install in conda environment /home/proj/bin/conda/envs/P_hermes
2021-06-03 14:19:35 hasta.scilifelab.se shipping.environment[21459] DEBUG Check if conda environment P_hermes exists
2021-06-03 14:19:35 hasta.scilifelab.se shipping.environment[21459] DEBUG Found conda env path: /home/proj/bin/conda/envs/P_shipping
2021-06-03 14:19:35 hasta.scilifelab.se shipping.environment[21459] INFO Locating base to /home/proj/bin/conda
2021-06-03 14:19:35 hasta.scilifelab.se shipping.environment[21459] DEBUG Conda env: P_hermes exists: True
2021-06-03 14:19:35 hasta.scilifelab.se shipping.environment[21459] DEBUG Found conda env path: /home/proj/bin/conda/envs/P_shipping
2021-06-03 14:19:35 hasta.scilifelab.se shipping.environment[21459] INFO Locating base to /home/proj/bin/conda
2021-06-03 14:19:35 hasta.scilifelab.se shipping.commands[21459] DEBUG Initialising Process with binary: /home/proj/bin/conda/envs/P_hermes/bin/python
2021-06-03 14:19:35 hasta.scilifelab.se shipping.commands[21459] DEBUG Use base call ['/home/proj/bin/conda/envs/P_hermes/bin/python']
2021-06-03 14:19:35 hasta.scilifelab.se shipping.commands[21459] INFO Running command /home/proj/bin/conda/envs/P_hermes/bin/python -m pip install -U cg-hermes
2021-06-03 14:19:39 hasta.scilifelab.se shipping.cli.deploy[21459] INFO Tool was successfully deployed
2021-06-03 14:19:39 hasta.scilifelab.se shipping.package[21459] INFO Fetch current version for package cg-hermes
2021-06-03 14:19:39 hasta.scilifelab.se shipping.commands[21459] INFO Running command /home/proj/bin/conda/envs/P_hermes/bin/python -m pip show cg-hermes
```

**Pull request merged**